TopQuarkAnalysis/TopEventProducers/interface/TopDecaySubset.h

```diff
@@ -43,12 +47,34 @@ class TopDecaySubset : public edm::EDProducer {
  private:
   /// find top quarks in list of input particles
   std::vector<const reco::GenParticle*> findTops(const reco::GenParticleCollection& parts);
+  /// find primal top quarks (top quarks from the hard interaction)
+  /// for Pythia6 this is identical to findDecayingTops
+  std::vector<const reco::GenParticle*> findPrimalTops(const reco::GenParticleCollection& parts);
```

Comment:
@kreczko 
Hello, 
If this findPrimalTops is identical to findTops for pythia6, we could simply replace the implementation of findTops with new one. In this way, it can be transparent for any analyzers who are using this function for both Run 1 and Run 2?
Maybe I missed something here.

Regards,
Taejeong